vgen/src/main/java/io/gapi/vgen/java/JavaContextCommon.java

```diff
@@ -212,7 +221,51 @@ public static Variable s_newVariable(
 
       public abstract Builder setParams(ImmutableList<Variable> params);
 
-      public Builder setParams(JavaGapicContext languageProvider, ImmutableList<Field> fields) {
+      public Builder setParams(JavaGapicContext languageProvider, Iterable<Field> fields) {
```

Comment:
`languageProvider` is no longer the correct name for the parameter - it should be `context`